app/components/containers/resizable-container.tsx

```
@@ -0,0 +1,88 @@
+        }
+    }, [storageKey]);
+
+    const startResizing = useCallback((e: React.MouseEvent) => {
+        e.preventDefault();
+        setIsResizing(true);
+    }, []);
+
+    const resize = useCallback((e: MouseEvent) => {
+        if (isResizing && containerRef.current) {
+            requestAnimationFrame(() => {
+                const rect = containerRef.current!.getBoundingClientRect();
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)